### PR TITLE
PUB-158: Comments from review

### PR DESCRIPTION
**Pull request by srp3rd**
I did a review and an edit pass through this, correcting some typos, adding footnotes, and rewording for clarity.

### CODE REVIEW

File: requirements.tex [hunk 1]
```diff
@@ -28,25 +28,25 @@ \subsection{Batch System} \label{sec:offlineprod}
 The batch system and associated interactive nodes are also used extensively by staff for development of future versions of the LSST Science Pipelines code.\cite{2019ASPC..523..521B}
 Though most science user access will be via the Data Access Center (DAC) (see \autoref{sec:usdac})
 there are requirements for science users to have some access to batch-type processing for larger-scale, non-interactive computations on the data.\cite{DMTN-223}
-This will be controlled by a Resource Allocation Committee
+This will be controlled by a Resource Allocation Committee.
 
 \subsection{Data transfer and preservation} \label{req:dbb}
 The USDF is responsible for operating the systems that track all raw data and released data products, including managing their movement, backup, and lifetime.
 These systems must be fault-tolerant and able to catch up after failures.
```
Comment (srp3rd) [inline requirements.tex]: I wasn't sure if you meant to say the fault-tolerant systems would catch up or that you were referring to "catch-up" mode to transfer files when there are network outages... or both

Comment (womullan) [reply requirements.tex]: reworded slightly

File: deploy.tex [hunk 2]
```diff
@@ -1,50 +1,48 @@
 \section{Continuous Deployment across the Rubin facilities} \label{sec:deploy}
 
-Being cloud ready is certainly the biggest challenge for most projects/applications which intend to leverage the commercial cloud.
-Early on we adopted Kubernetes which provided a service architectures that is well isolated from the underlying infrastructure.
+Being cloud-ready is certainly the biggest challenge for most projects/applications which intend to leverage the commercial cloud.
```
Comment (womullan) [inline deploy.tex:3]: hyphenation a real divisive device I usually avoid completely, I suppose in US it is more popular.

File: deploy.tex [hunk 3]
```diff
@@ -1,50 +1,48 @@
...
-Each thus provides a kubernetes environment with disk, network and compute resources upon which we spin up our services.
-A vault for secrets is also needed.
+Our agreement with each facility is to provide a Kubernetes deployment platform upon which we may deploy our services and our secrets vault for security.
+Each thus provides a Kubernetes environment with disk, network and compute resources upon which we spin up our services.
 From the service perspective each facility looks similar.
 It feels like the first time that this actually works well - we have learned to make our applications portable of course but also the technology is so much better than any precursors.
 
 \subsection{Phalanx, Helm and ArgoCD}
 
-Helm\footnote{https://helm.sh/} is a standard approach to tell Kubernetes about software.
```
Comment (womullan) [inline deploy.tex:26]: I wonder if this should stay here and remove it from the earlier sect
ion so all the tech footnotes are together.

**Review by womullan — APPROVED** [review]

**Pull request merged**